Automattic/wp-calypso
# Unified Blaze illustration

Related to https://github.com/Automattic/wp-calypso/issues/72732

## Proposed Changes

* Unified Blaze illustration by removing the old `blaze-drop-down.svg`

## Testing Instructions



* Access /marketing/traffic and make sure we have the updated Blaze illustration:

<img width="794" alt="image" src="https://user-images.githubusercontent.com/1044309/224714692-891e6642-94d1-4aa7-90f0-1dad0573adac.png">


* Set `showBlazePromo = true` at client/my-sites/stats/promo-cards/index.jsx and make sure /stats page also uses the updated version of Blaze illustration:

<img width="801" alt="image" src="https://user-images.githubusercontent.com/1044309/224715208-c7013b84-847e-4126-a09c-296e88aecb0b.png">


## Pre-merge Checklist
